shell/edit/logging.banzaicloud.io.output/index.vue

```diff
@@ -40,8 +47,6 @@ export default {
     }));
 
     if (this.mode !== _VIEW) {
-      this.$set(this.value, 'spec', this.value.spec || {});
```

Comment:
How come this was removed?

Reply:
It's a duplicate in relation to https://github.com/rancher/dashboard/blob/master/shell/edit/logging.banzaicloud.io.output/index.vue#L34. I kept this one.